### PR TITLE
Added separate docker volume for VPN config

### PR DESCRIPTION
**Pull request by bartvanb**
First functional version where algorithms can connect to one another

### CODE REVIEW

File: vantage6/cli/node.py [hunk 1]
```diff
@@ -309,13 +309,16 @@ def cli_node_start(name, config, environment, system_folders, image, keep,
     info("Creating Docker data volume")
     data_volume = docker_client.volumes.create(
         f"{ctx.docker_container_name}-vol")
+    vpn_volume = docker_client.volumes.create(
+        f"{ctx.docker_container_name}-vpn-vol")
```
Comment (frankcorneliusmartin) [inline vantage6/cli/node.py]: It would be nicer to use the new `ctx.docker_vpn_volume_name` property here.

File: vantage6/cli/node.py [hunk 2]
```diff
@@ -309,13 +309,16 @@ def cli_node_start(name, config, environment, system_folders, image, keep,
     info("Creating Docker data volume")
     data_volume = docker_client.volumes.create(
         f"{ctx.docker_container_name}-vol")
```
Comment (frankcorneliusmartin) [inline vantage6/cli/node.py]: If you are already at it.. change this one too: 
`ctx.docker_volume_name`

**Review by frankcorneliusmartin — CHANGES_REQUESTED** [review]
2 minor changes

File: vantage6/cli/node.py (hunk 1 above)
Comment (bartvanb) [reply vantage6/cli/node.py]: Done

File: vantage6/cli/node.py (hunk 2 above)
Comment (bartvanb) [reply vantage6/cli/node.py]: Done

**Review by frankcorneliusmartin — APPROVED** [review]

**Pull request merged**